src/getting-started.handlebars

```diff
@@ -58,7 +58,10 @@
                       <p>So now you have your client id and secret, you are ready to get a token.</p>
                       <p>Concatenate your client id and secret with a <code>:</code> in between. For example, with the client id and secret given as example in step 1, this will give:</p>
                       <pre class="hljs"><code>3e2iqilq2ygwk0ccgogkcwco8oosckkkk4gkoc0k4s8s044wss<span class="hljs-string">:</span>44ectenmudus8g88w4wkws84044ckw0k4w4kg0sokoss84oko8</code></pre>
-                      <p>Then encode this string in base 64. They are utilities functions in every languages to do so and even sites, like this <a href="https://www.base64encode.org/">one</a> for example. You will get:</p>
+                      <p>Then encode this string in base 64. They are utilities functions in every languages to do so and even sites, like this <a href="https://www.base64encode.org/">one</a> for example.</p>
+                      <p>You can also use the terminal to get the base64 encoded string:</p>
+                      <pre class="hljs"><code>echo 3e2iqilq2ygwk0ccgogkcwco8oosckkkk4gkoc0k4s8s044wss:44ectenmudus8g88w4wkws84044ckw0k4w4kg0sokoss84oko8 | base64</code></pre>
+                      <p>You will get:</p>
```

Comment:
@skeleton maybe because base64 is not implemented the same way in [MacOS X](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/base64.1.html) and [GNU/Linux](https://linux.die.net/man/1/base64)? I can see they don't rely on the same RFCs o_O